cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -2962,7 +2953,7 @@ bool CacheAllocator<CacheTrait>::markMovingForSlabRelease(
     itemFreed = true;
 
     if (shutDownInProgress_) {
-      XDCHECK(!static_cast<Item*>(alloc)->isMoving());
+      XDCHECK(!static_cast<Item*>(alloc)->isExclusive());
```

Comment:
Should we remove this assert? I think this can be true now if an item is marked as exclusive by eviction thread while the slab release gets to this line.

Reply:
Yes, you are right. Removed.